modules/client/src/tokenVoting/internal/client/methods.ts

```diff
@@ -417,6 +422,25 @@ export class TokenVotingClientMethods extends ClientCore
     };
   }
 
+  public async *delegateTokens(
```

Comment:
I would also add `undelegateTokens()` basically calling `delegateTokens(tokenContract, myself)`

I would also expose a way to call `delegates()`, to know if we are delegating to someone or not